core/cloudflow-spark/src/main/scala/cloudflow/spark/SparkStreamlet.scala

```diff
@@ -106,8 +106,9 @@ trait SparkStreamlet extends Streamlet[SparkStreamletContext] with Serializable
           // stop all query execution
           streamletQueryExecution.stop()
 
-        } else ()
+        }
       }
+      val done = system.scheduler.scheduleWithFixedDelay(InitialDelay, MonitorFrequency)(runnable)
```

Comment:
Just curious why you made this change. Readability?

Reply:
deprecation